tests/installation/partitioning_raid.pm

```diff
@@ -196,32 +207,56 @@ sub run() {
 
         # in last step of for loop edit first vda1 and format it as EFI ESP, preparation for fate#322485
         if ($_ == 4 and get_var('UEFI')) {
+            wait_idle 3;
```

Comment:
That is what I was trying to say. I don't expect them to be used at the end. There are only there until I covered OFW and UEFI. The PR is in status 'not ready'